### DISCUSSION TITLE
Only have a few days left… uh oh!

### DISCUSSION THREAD

**Discussion by Jacob1863** [ppy/osu#29883] [Q&A] [answered]
My crappy Android device doesn’t wanna run lazer, and I had no choice but to use iOS as an option. But now this is almost an end of an era.
![IMG_3354](https://github.com/user-attachments/assets/499c3f28-07c2-4e09-bfa7-e7b89d8af244)

**Comment by peppy** [comment, marked as answer]
smoogi is hard at work trying to fix this in time.

**Reply by Jacob1863** [reply]
Alright peppy, thanks for informing me!
Smoogi, if you’re reading this. Good luck!
